plugins/sentry/src/components/useProjectSlug.ts

```diff
@@ -18,6 +18,13 @@ import { Entity } from '@backstage/catalog-model';
 
 export const SENTRY_PROJECT_SLUG_ANNOTATION = 'sentry.io/project-slug';
 
+/**
+ * @public
+ * Checks to see if sentry is available
+ */
+export const isSentryAvailable = (entity: Entity) =>
```

Comment:
@jhaals would you be able to advise on where you feel it would be easier to discover?  I'm looking through different plugins and it seems that the `is(Plugin)Available` is located in a number of different places.  Some plugins include the check for being available within the `plugin.ts`, others have it within the `index.ts`, and some are within either the Router or component itself. 

Since I was starting to model this after the SonarQube plugin which includes the `isSonarQubeAvailable` within the `useProjectKeys.ts` file, this is why I thought the `useProjectSlug.ts` would be ideal.

Reply:
Existing plugins are necessarily not doing the most ideal package split up. Thanks for the update!